packages/backend/src/api/controllers/ForcedTradeOfferController.ts

```diff
@@ -26,6 +28,17 @@ export class ForcedTradeOfferController {
     private userRegistrationEventRepository: UserRegistrationEventRepository
   ) {}
 
+  async getOffersIndexPage(
+    account: EthereumAddress | undefined
+  ): Promise<ControllerResult> {
+    const offers = await this.offerRepository.getAllInitialOffers()
```

Comment:
We probably want to paginate this (and filter by type and asset), but this PR is about the HTML, so I'm just leaving it here as a note.